frontend/src/pages/IntegrationsPage/components/GitHubIntegration/utils.ts

```diff
@@ -38,14 +38,8 @@ export const getGitHubInstallationOAuthUrl = (
 	projectId: string,
 	workspaceId: string,
 ) => {
-	let redirectPath = window.location.pathname
-	if (redirectPath.length > 3) {
-		// remove project_id and prepended slash
-		redirectPath = redirectPath.substring(redirectPath.indexOf('/', 1) + 1)
```

Comment:
The `next` attribute was not being used for GitHub integrations previously, but believe we want to remove this. With it, the next url was invalid and redirected users to sessions.

Reply:
Is this still being used? @Vadman97